frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseSlideOutBranchAction.java

```diff
@@ -59,22 +50,20 @@ public BaseSlideOutBranchAction() {}
 
   @UIEffect
   public void doSlideOut(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch branchToSlideOut) {
-    LOG.debug(() -> "Entering: anActionEvent = ${anActionEvent}, " +
-        "branchToSlideOut = ${branchToSlideOut} (${branchToSlideOut.getName()})");
-    Project project = anActionEvent.getProject();
-    assert project != null : "Can't get project from anActionEvent variable";
-
-    var branchLayout = anActionEvent.getData(DataKeys.KEY_BRANCH_LAYOUT);
-    assert branchLayout != null : "Can't get branch layout";
-
-    var branchName = branchToSlideOut.getName();
+    LOG.debug(() -> "Entering: branchToSlideOut = ${branchToSlideOut}");
+    String branchName = branchToSlideOut.getName();
+    Project project = ActionUtils.getProject(anActionEvent);
+    var branchLayout = ActionUtils.getBranchLayout(anActionEvent);
+    var gitMacheteFilePath = ActionUtils.getGitMacheteFilePath(anActionEvent);
+    if (branchLayout.isEmpty() || gitMacheteFilePath.isEmpty()) {
+      return;
```

Comment:
Maybe some log or even noticeable message for user in that cases? The same for other actions.

Reply:
Hmm TBH I'm reluctant to pollute our action codebase with more code for handling extraordinary situations - I'd realistically expect any of this keys to be empty only in case of some nasty race conditions. I don't even think we should bother the user, but rather let them click again even it happens.

Reply:
But if it comes to some WARN-level logging unnoticable for user - I think it's okay, I'm gonna add that now